### PR TITLE
corrected and added test for ResizeFormListener

### PR DESCRIPTION
**Pull request by ahmetakbn**
### Subject

Improving tests for `SonataCoreBundle`

### CODE REVIEW

File: Tests/Form/EventListener/ResizeFormListenerTest.php [hunk 1]
```diff
@@ -13,27 +13,309 @@
 
 use Sonata\CoreBundle\Form\EventListener\ResizeFormListener;
 use Symfony\Component\Form\FormEvent;
+use Symfony\Component\Form\FormEvents;
+use Symfony\Component\PropertyAccess\Tests\Fixtures\TraversableArrayObject;
 
 /**
- * @author Hugo Briand <briand@ekino.com>
```
Comment (core23) [inline Tests/Form/EventListener/ResizeFormListenerTest.php]: Why did you remove this?

Comment (ahmetakbn) [inline Tests/Form/EventListener/ResizeFormListenerTest.php]: Because there was one test method in this class and it was testing nothing :)

Comment (core23) [inline Tests/Form/EventListener/ResizeFormListenerTest.php]: The diff is pretty ugly... but you're right. In this case we could remove it.

Comment (ahmetakbn) [inline Tests/Form/EventListener/ResizeFormListenerTest.php]: Yes pretty ugly, actually it was a method and I removed it.

**Comment by OskarStark** [comment]
LGTM 👍 

ping @greg0ire

**Comment by greg0ire** [comment]
I'm on holidays until the 17th @Oskarstark please ask someone else :)

**Comment by Soullivaneuh** [comment]
Thank you @ahmetakbn
